### PR TITLE
Update readme and migration guide

### PR DESCRIPTION
**Pull request by cah-brian-gantzler**
While a work in progress doesnt hurt to get some of it out there. Will be updating more.

### CODE REVIEW

File: docs/migration.md [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+    
+    @hasMany('product', { async: true, inverse: null }),
+    products;
+}
+```
+would look like this in MirageJS. See the mirage guides if you have any questions on 
+defining models [MirageJS Models](https://miragejs.com/docs/main-concepts/models/)
+
+```js  
+//mirage/models/accoutn.js
```
Comment (NullVoxPopuli) [inline docs/migration.md:48]:
```suggestion
//mirage/models/account.js
```

File: docs/migration.md [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+  let finalConfig = {
+    ...config,
+    models: { ...discoverEmberDataModels(), ...config.models },
+    routes,
+  };
+
+  return createServer(finalConfig);
+}
+```
+It will not look like this
```
Comment (NullVoxPopuli) [inline docs/migration.md:69]:
```suggestion
It will now look like this
```

Comment (cah-brian-gantzler) [reply docs/migration.md]: fixed. Just now saw that "commit suggestion" is an option. Will have to try that sometime

File: docs/migration.md (hunk 1 above)
Comment (cah-brian-gantzler) [reply docs/migration.md]: fixed

**Pull request closed without merging**